### PR TITLE
Change modal arrangement, add requirements

### PR DESCRIPTION
**Pull request by inquisitiv-e**
- Item modal fixed to center position
- Making every field required. Submit button disabled until all fields entered.
- Fix the location drop-down
- Make price input prettier
Screen Shot 2019-07-05 at 1 33 05 AM

### CODE REVIEW

File: imports/ui/components/FreshForm.jsx [hunk 1]
```diff
@@ -33,8 +36,9 @@ class FreshForm extends Component {
     this.setState({ price: event.target.value });
   };
 
-  handleChangeUnit = (event, { value }) => {
+  handleChangeUnit = value => {
     this.setState({ unit: value });
+    console.log(value);
```
Comment (YaoLiu93) [inline imports/ui/components/FreshForm.jsx]: Remove console log

File: imports/ui/components/FreshForm.jsx [hunk 2]
```diff
@@ -45,6 +49,14 @@ class FreshForm extends Component {
   };
 
   handleSubmit = event => {
+    event.preventDefault();
+    !this.state.add || !this.state.price || !this.state.name
```
Comment (nicolehli) [inline imports/ui/components/FreshForm.jsx]: What happens when we put in somewhere for location but not select something from autofill (then we have address but no lat lon). Is that okay? Will map cards be affected? Do we need check on lat lon? Just curious

Comment (nicolehli) [reply imports/ui/components/FreshForm.jsx]: Also, modal looking great

Comment (inquisitiv-e) [reply imports/ui/components/FreshForm.jsx]: You must make a select by clicking the address :) Otherwise it won't let you submit

Comment (nicolehli) [reply imports/ui/components/FreshForm.jsx]: Actually, I was able to submit without clicking an address from dropdown, just put in some text in address input. So the card does not have lat lon and the submitted card has no address.

Comment (inquisitiv-e) [reply imports/ui/components/FreshForm.jsx]: It doesn't work for me.... 
Screen Shot 2019-07-06 at 12 15 41 AM

Comment (nicolehli) [reply imports/ui/components/FreshForm.jsx]: image

I was able to reproduce it. I think you just have to press 'Enter' Key and submit shows up enabled.

Comment (inquisitiv-e) [reply imports/ui/components/FreshForm.jsx]: @nicolehli Ohh I see! Yeah upon pressing enter I get the same issue. I'll add the lat & lng check!!!

Comment (inquisitiv-e) [reply imports/ui/components/FreshForm.jsx]: Fixed in the new commit

**Comment by inquisitiv-e** [comment]
#59

**Pull request merged**